chall07/mlink.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+import sys
+
+rows = len(sys.argv) - 1
+array = []
+newarray = []
+
+if 1 <= rows <= 20:
+
+    for i in range(1, rows + 1):
+        if len(sys.argv[i]) == rows:
+            s = sys.argv[i]
+            for j in range(rows):
+                if s[j] > '9' or s[j] < '0':
+                    print(f"usage: {sys.argv[0]} <1-9 squared_rows...>")
+                    exit() 
```

Comment:
instead of print + exit, you can directly give the message into the exit argument. 

exit("message")